frontend/package.json

```diff
@@ -26,6 +26,7 @@
     "react": "16.12.0",
     "react-app-polyfill": "1.0.5",
     "react-bootstrap": "1.0.0-beta.16",
+    "react-click-away-listener": "^1.1.0",
```

Comment:
/nit: please remove the `^` - we want dependencies locked in to avoid breaking builds when they get updated without our review. thanks!

Reply:
Also, since we are replacing `ag-grid` - wouldn't it make sense to remove it from `package.json`? I still see it in lines 11-12 above...

```json
    "ag-grid-community": "23.0.2",
    "ag-grid-react": "23.0.3",
```